packages/terra-text/docs/text-colors.md

```diff
@@ -1,3 +1,28 @@
 ## Text Colors
 
-Using the `color` prop, authors can set the text color of any `<Text />` component using a valid CSS color value.
+By passing in a className, authors can manually set the text color of any `<Text />` component.
+
+```css
+:root {
+  --terra-text-attention-color: rgb(220, 20, 60);
```

Comment:
The `--terra` prefix is intended to be used only by components we write in terra. Since this example is intended for people to write in their own app, we should avoid using the `--terra` prefix. I'd recommend dropping the prefix or use a prefix like `--my-app`.

Reply:
As per other comment, have renamed the docs to not use `--terra` but left the colors.css in the examples along, here https://github.com/cerner/terra-core/pull/2211/commits/0cb2f32c1a47112a45feb6c38ab703cbc70c40ca